avpipe.h

```diff
@@ -111,6 +111,50 @@ probe(
     xcprobe_t **xcprobe,
     int *n_streams);
 
+/**
+ * @brief   Get count of dictionary keys.
+ *
+ * @param   dict    Dictionary pointer.
+ * @return  Count of dictionary keys.
+ */
+int
+dict_count(
```

Comment:
avpipe.h includes the avpipe interface functions and these dict functions should not be exposed here.

Do we need these functions to be defined here?